utrc-cms/static/utrc-cms/css/src/placeholder.css

```diff
@@ -0,0 +1,3 @@
+/* DO NOT ADD STYLES HERE; ONLY IMPORT OTHER STYLESHEETS */
+
+/* At least one stylesheet must exist until CSS build does not assume such */
```

Comment:
**Gross!**

The CSS build process assumes at least one stylesheet for every project. I have finally created a ticket to solve this*: https://github.com/TACC/Core-CMS/issues/350

> \* Because sites have begun to not need custom assets.